lib/tbd/geo.rb

```diff
@@ -847,8 +843,9 @@ def kiva(model = nil, walls = {}, floors = {}, edges = {})
           next unless walls[i][:boundary].downcase == "foundation"
           next     if walls[i].key?(:kiva)
 
-          floors[id][:kiva] = :basement
-          walls[i  ][:kiva] = id
+          floors[id][:kiva   ]  = :basement
+          floors[id][:exposed] += edge[:length]
```

Comment:
1 of 2 lines where "Foundation"-facing wall/slab edge lengths get tallied as exposed foundation perimeter.